streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -1066,7 +1063,7 @@ public String toString(final String indent) {
         return sb.toString();
     }
 
-    private class StreamsMetricsImpl implements StreamsMetrics, ThreadCacheMetrics {
+    static class StreamsMetricsImpl implements StreamsMetrics, ThreadCacheMetrics {
```

Comment:
This is not about this PR: in #1446 we are removing specific cache sensors since we are adding sensors for generic purposes already. There will be some major conflicts between these two.